src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPool.cs

```diff
@@ -736,6 +749,19 @@ public bool CleanCacheAndDisposeIfUnused()
             return false;
         }
 
+        /// <summary>Gets whether we're running on Windows 7 or Windows 2008 R2.</summary>
+        private static bool GetIsWindows7Or2008R2()
```

Comment:
nit:  Is this name recommended by .NET Design guideline?  IMHO, I think since it returns a bool, it should simply be called "IsWindows7Or2008R2".

Reply:
In general, I agree. In this case, I didn't want calling code to access it directly, instead preferring to use the s_isWindows7Or2008R2 field that's initialized to the result of calling this method.  So I chose to go with a name that sounded a bit less appealing.